driver/src/test/unit/com/mongodb/util/JSONCallbackTest.java

```diff
@@ -139,4 +142,45 @@ public void numberDecimalParsing() {
         Decimal128 number = (Decimal128) JSON.parse(("{\"$numberDecimal\" : \"314E-2\"}"));
         assertEquals(number, Decimal128.parse("314E-2"));
     }
+
+    private Object objectDoneTestHelper(final String jsonString, final String rootName) {
```

Comment:
I've extended this method now, but the 128 int needed to be cast to a byte:

`assertEquals((byte) 128, parsedBinaryWithHexType.getType());`